src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -2013,5 +2014,19 @@ public async Task SendAsync_InvalidRequestUri_Throws()
             request = new HttpRequestMessage(HttpMethod.Get, new Uri("foo://foo.bar"));
             await Assert.ThrowsAsync<NotSupportedException>(() => invoker.SendAsync(request, CancellationToken.None));
         }
+
+        [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void AppContextSetData_SetDefaultMaxConnectionsPerServer(bool asInt)
+        {
+            RemoteExecutor.Invoke((asInt) =>
+            {
+                object data = asInt == "true" ? 1 : "1";
```

Comment:
`true.ToString() == "True"`
```suggestion
                object data = asInt == "True" ? 1 : "1";
```

or

```suggestion
                object data = asInt == Boolean.TrueString ? 1 : "1";
```